docs/dictionary/function/machine.lcdoc

```diff
@@ -30,22 +31,24 @@ The <machine> <function> <return|returns> a <string>.
 -   iPhone Simulator: the device is a simulated iPhone
 -   iPad: the device is the iPad
 -   iPad Simulator: the device is a simulator iPad
-
+-   : macOS, Unix and Android returns described below
```

Comment:
I like this and I agree. It's interesting about FM64 too. Would you suggest FM64 being among those platform paragraphs with the x86_64 response?